subsys/bluetooth/services/fast_pair/fp_keys.c

```diff
@@ -423,17 +476,29 @@ void fp_keys_bt_auth_progress(const struct bt_conn *conn, bool authenticated)
 
 void fp_keys_drop_key(const struct bt_conn *conn)
 {
+	if (!is_enabled) {
```

Comment:
In case of init or deinit fail, some modules may stay active and call other modules's API. I will see how will it look if all modules will depend on `bt_fast_pair_is_ready` instead of their private `is_enabled`.

Reply:
If we may end up with a partially initialized state and want to allow calling `enable/disable` API in such state, keeping information about module state may become unavoidable for some modules